lib/services/execution_iframe.dart

```diff
@@ -79,20 +79,20 @@ class ExecutionServiceIFrame implements ExecutionService {
     _frameSrc = src;
   }
 
-  /// TODO(redbrogdon): Format message so internal double quotes are escaped.
   @override
   String get testResultDecoration => '''
 void _result(bool success, [List<String> messages = const []]) {
   // Join messages into a comma-separated list for inclusion in the JSON array.
-  final joinedMessages = messages.map((m) => '"\$m"').join(',');
+  final joinedMessages = 
```

Comment:
Can we add some form of testing to this?

Reply:
Definitely a good idea, especially since the double escaping is a little tricky to follow. I'll write something up!

Reply:
I got some sort of test working, not 100% sure it's the best way to handle it though: https://github.com/dart-lang/dart-pad/pull/2474#issuecomment-1464589363